core/src/main/resources/filodb-defaults.conf

```diff
@@ -338,3 +338,53 @@ kamon.zipkin {
   max.requests = 128
   message.max.bytes = 262144
 }
+
+kamon {
+  prometheus.buckets {
+    # Have more buckets, better resolution really helps esp with heatmaps
+    default-buckets = [
+      4,
+      8,
+      16,
+      32,
+      64,
+      128,
+      256,
+      512,
+      1024,
+      2048,
+      4096,
+      8192,
+      16384,
+      32768,
+      65536,
+      128000,
+      256000,
+      512000
+    ]
+
+    # Start at 0.1ms so we can measure smaller timings
+    time-buckets = [
+      0.0001,
+      0.00025,
+      0.0005,
+      0.001,
+      0.0025,
+      0.005,
+      0.01,
+      0.025,
+      0.05,
+      0.1,
+      0.25,
+      0.5,
+      1,
```

Comment:
These buckets don't seem to have uniform ratios.

Reply:
It's OK to not have uniform ratios.  We can make them uniform I guess for slightly better distribution, ultimately it's what makes sense to us.

Reply:
What makes "sense" is bit subjective. We don't have any data to know what the "best" defaults are. My thought is that the ratios should at least be uniform, so it seems more obvious that the defaults aren't that well thought out. The numbers aren't as magical.